deploy/charts/openebs-monitoring/templates/podmonitors.yaml

```diff
@@ -4,7 +4,7 @@
 apiVersion: monitoring.coreos.com/v1
 kind: PodMonitor
 metadata:
-  name: {{ template "openebs-monitoring.fullname" $ }}-{{ $openebsEngine }}
+  name: {{ lower (printf "%s-%s" (include "openebs-monitoring.fullname" $) $openebsEngine) | trunc 63 | trimSuffix "-" }}
```

Comment:
To convert metadata.name to lowercase otherwise, helm installation would give an error if openebsEngine are defined in uppercase letters.